Week3/homework/step4-bonus.js

```diff
@@ -2,9 +2,12 @@
 
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
-// Add your function here. Try and come up with a good name for this function
+function makeItUnique(arr) {
+  return Array.from(new Set(arr));
+}
+
 
 // Replace `yourFunction` with the name of the function you just created
-const uniqueValues = yourFunction(values);
+const uniqueValues = makeItUnique(values);
 
```

Comment:
✅ Very nice use of making different datatypes work for you.  Of course, the `Set` did not exist in Javascript a few years ago, and you had to do it another way, usually using a filter.